doc/cabal-package.rst

```diff
@@ -3152,7 +3152,7 @@ the configured data directory for ``pretty-show`` is controlled with the
 Accessing the package version
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-The aforementioned auto generated :file:`Paths_{pkgname}` module also
+The aforementioned auto generated :file:`PackageInfos_{pkgname}` module also
```

Comment:
I suppose it's not "aforementioned" any more? And it doesn't export "also", but just exports?